manifests/base/kustomization.yaml

```diff
@@ -193,3 +193,180 @@ vars:
       fieldpath: data.odh-rstudio-gpu-notebook-n-1
 configurations:
   - params.yaml
+
+configMapGenerator:
+  - name: notebook
```

Comment:
As we have another `configmapgenerator` in this file, line 20.
probably moving this under there would be more cleaner